great_expectations/core/validation_definition.py

```diff
@@ -104,13 +102,6 @@ class Config:
     data: BatchDefinition
     suite: ExpectationSuite
     id: Union[str, None] = None
-    _validation_results_store: ValidationResultsStore = PrivateAttr()
-
-    def __init__(self, **data: Any):
-        super().__init__(**data)
-
-        # TODO: Migrate this to model_post_init when we get to pydantic 2
-        self._validation_results_store = project_manager.get_validation_results_store()
```

Comment:
Other parts of our codebase will just inline import as we see fit - we should do this consistently